packages/rich-text/src/plugins/Table/useAnchorNode.ts

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```

Comment:
Don't we need to do `import * as React from 'react';` in order to use `React.useEffect`?

More a curiosity from my side :)

Reply:
No need for `import * as React` really. I think that was a problem a few years back but not anymore. Can't remember exactly whether it's the `esModuleInterop` flag in `tsconfig.json` that does the trick or something else.

Anyway, I changed the import to `{ useState, useEffect }` since I don't need `React` anyway.

Reply:
Btw, looks like `import * as React` is [more future proof](https://twitter.com/dan_abramov/status/1308739731551858689?s=20) than `import React`

Reply:
On the other hand, we don't need to import `React` anyway when migrate to React v17 :)